src/main/java/mate/academy/spring/model/User.java

```diff
@@ -41,10 +50,20 @@ public void setPassword(String password) {
         this.password = password;
     }
 
+    public Set<Role> getRoles() {
+        return roles;
+    }
+
+    public void setRoles(Set<Role> roles) {
+        this.roles = roles;
+    }
+
     @Override
     public String toString() {
         return "User{"
-                + "id=" + id
-                + ", email='" + email + '\'' + '}';
+                + "email='" + email + '\''
+                + ", password='" + password + '\''
```

Comment:
I dont think we need to include password into ToString ,method, it is sensitive data